src/services/prisma-datastore/schema.prisma

```diff
@@ -180,6 +180,7 @@ model Reading {
 
   latest            Sensor?         @relation(name: "latest")
 
+  @@unique([sensor_id, when, value])    // natural key
```

Comment:
Should the value be part of the natural key?

Reply:
I was imagining a case something like this:
```
Time: Wednesday,  Sensor: Opinion,   Value: { person: Carl, opinion: desirable }
Time: Wednesday,  Sensor: Opinion,   Value: { person: Mat, opinion: questionable }
```

Reply:
better example above